pkg/upgradeservice/handlers/config.go

```diff
@@ -411,8 +412,12 @@ func (h *Handler) SaveAppConfig(w http.ResponseWriter, r *http.Request) {
 
 	reportingFn := func() error {
 		if params.AppIsAirgap {
-			// TODO NOW: airgap reporting
-			return nil
+			clientset, err := k8sutil.GetClientset()
+			if err != nil {
+				return errors.Wrap(err, "failed to get clientset")
+			}
+			report := reporting.BuildInstanceReport(appLicense.Spec.LicenseID, params.ReportingInfo)
+			return reporting.AppendReport(clientset, util.PodNamespace, app.Slug, report)
```

Comment:
how about creating a SubmitAppInfo in the upgradeservice package that does this:
```go
		if params.AppIsAirgap {
			clientset, err := k8sutil.GetClientset()
			if err != nil {
				return errors.Wrap(err, "failed to get clientset")
			}
			report := reporting.BuildInstanceReport(appLicense.Spec.LicenseID, params.ReportingInfo)
			return reporting.AppendReport(clientset, util.PodNamespace, app.Slug, report)
		}
		return reporting.SendOnlineAppInfo(appLicense, params.ReportingInfo)
```
so we can use it in other places later?